File: ServerFacade/src/main/java/Request/GameRequest.java
package Request;

public class GameRequest {
  private String username;
  private String location;

  public GameRequest(String username, String location) {
    this.username = username;
    this.location = location;
  }

  public String getUsername() {
    return username;
  }

  public String getLocation() {
    return location;
  }
}

File: FrontEnd/lib/Request/GameRequest.dart
import 'package:json_annotation/json_annotation.dart';

part 'GameRequest.g.dart';

@JsonSerializable()

class GameRequest {
  String username;
  String location;

  GameRequest(this.username, this.location);

  factory GameRequest.fromJson(Map<String, dynamic> json) => _$GameRequestFromJson(json);

  Map<String, dynamic> toJson() => _$GameRequestToJson(this);
}
